### ISSUE TITLE
Custom atlantis.yaml functionality removed

### ISSUE THREAD

**Issue by Crevil** [cloudposse/atlantis#35]
I'm looking into having custom `atlantis.yaml` configs for different servers and found this fork through a lengthy discussion on the topic in the upstream (https://github.com/runatlantis/atlantis/pull/310#issuecomment-432908203).

This fork looked promising and I've made a POC using this but just found out that PR #22 actually removed the functionality implemented in #12.

Have you found another way of dealing with this or was this some kind of mistake with the upstream merge?

**Comment by aknysh** [comment]
@Crevil 
we experimented with a lot of new features, but in the end ended up just adding our GitHub team whitelist feature to our fork for security.
We added a few more features before, but once we got the latest code from the upstream, we just added `gh-team-whitelist` to it (so I would say removing the other features was on purpose since we wanted to maintain as few differences from the upstream as possible).
But you are welcome to add the features from our PRs to your own fork.
